docs/src/main/sphinx/release/release-452.md

```diff
@@ -0,0 +1,47 @@
+# Release 452 (3 Jul 2024)
+
+## General
+
+* Fix query plans occasionally not rendering the stage details page. ({issue}`22542`)
+* Fix a performance regression when using the native filesystem for Azure. ({issue}`22561`)
+
+## ClickHouse connector
+
+* Add `query` table function for full query pass-through to the connector. ({issue}`16182`)
+
+## Delta Lake connector
+
+* Add support for type coercion when adding new columns. ({issue}`19708`)
+* Improve performance of reading from Parquet files with large schemas. ({issue}`22434`)
+* Fix incorrect results when reading `INT32` values in Parquet files. ({issue}`21556`)
+
+## Hive connector
+
+* Add support for coercion of structural data types for non-partitioned ORC
```

Comment:
Do we use the term "ORC tables" .. or do we need to expand to "tables using ORC files"